src/components/skeleton/Header/Header.js

```diff
@@ -57,4 +60,9 @@ export default class Header extends Component {
     // (solution: put some sort of icon at the top of the navigation so its non-clickabel?:)
     setTimeout(() => this.setState({isNavOpening: false}), 1000);
   }
+
+  _reverseName(name) {
+    var nameInParts = name.split(' ');
+    return nameInParts[1] + ' ' + nameInParts[0];
+  }
```

Comment:
Dan dan nog een bug om te fixen of we laten de reverse dan is het achternaam + voornaam (recht vanuit DB) bv. Van Schandevijl Wouter. Das een keuze e